resources/assets/sass/components/_list.scss

```diff
@@ -69,6 +83,25 @@
             }
         }
 
+            .c-list__item-icon {
+
+                display: inline-block;
+                margin-right: $spacer / 2;
+                height: $spacer * 1.5;
+                width: $spacer * 1.5;
+
+                svg {
+
+                    .c-list.m-default & {
+
+                        fill: $color-text-link;
+                    }
+
+                    height: 100%;
```

Comment:
Expected item on line 100 to appear before line 95. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets